### ISSUE TITLE
Making Mii's have a baby + unlocking all food, clothes, special foods, hats and interiors corrupts save file 

### ISSUE THREAD

**Issue by LitPotato2895** [Brionjv/Tomodachi-Life-Save-Editor#118]
so i set my Mii's status to married soulmates, i have babies on and i respected all conditions and they still wont have a baby :(

and for the unlocking issue im having, i have all buildings unlocked and it still wont work

Im using a regular 3DS, JKSV save editor and editor version 4.0.2, is this a bug or is it something im doing?

**Comment by dreamycarnival** [comment]
Seems like it's always a problem no matter the version. I've been trying to force a couple I want to be friends on my island since RNG's a fucking piece of shit, but it ends up corrupting the save data or spawning the wrong problem. It's bizarre, and I wish the creator would fix this bug.

**Comment by LitPotato2895** [comment]
> Seems like it's always a problem no matter the version. I've been trying to force a couple I want to be friends on my island since RNG's a fucking piece of shit, but it ends up corrupting the save data or spawning the wrong problem. It's bizarre, and I wish the creator would fix this bug.

yeah :( i hope it gets fixed in the next update, but i'll cut the dev some slack, coding's hard at times

**Comment by Brionjv** [comment]
I'll try to fix that

**Comment by LitPotato2895** [comment]
> I'll try to fix that

thank you :)

**Comment by LitPotato2895** [comment]
> I'll try to fix that

forcing a baby still corrupts it and so does any other interaction (not mad)

**Comment by Brionjv** [comment]
I do nothing for now, I need to do some tests

**Comment by LitPotato2895** [comment]
> I do nothing for now, I need to do some tests

alrighty, good luck

**Comment by Kagura101** [comment]
Hey, even in the 4.0.4 version, forcing any interaction corrupts the save :(
